db/migrate/20131210202928_update_log_collection_path_in_configurations_settings.rb

```diff
@@ -4,8 +4,8 @@ class Configuration < ActiveRecord::Base
     self.inheritance_column = :_type_disabled # disable STI
   end
 
-  NEW_PG_CONF_PATH = "/opt/rh/postgresql92/root/var/lib/pgsql/data/*.conf"
-  NEW_PG_LOGS_PATH = "/opt/rh/postgresql92/root/var/lib/pgsql/data/pg_log/*"
+  NEW_PG_CONF_PATH = "/opt/rh/rh-postgresql94/root/var/lib/pgsql/data/*.conf"
+  NEW_PG_LOGS_PATH = "/opt/rh/rh-postgresql94/root/var/lib/pgsql/data/pg_log/*"
```

Comment:
Yes, it would be nice to change the log collection code (the consumer of this configuration setting) to get this information in one place, like a class constant or something, but not in the configuration itself.  I guess I can revisit this when it's closer to ready and either add a new migration to convert postgresql92 to rh-postgresql94 or see about externalizing the setting outside of the configuration itself.  I don't know that it makes sense to even externalize these settings here.